### ISSUE TITLE
Why the mean test accuracy increases slow using Fedavg

### ISSUE THREAD

**Issue by Sherrylife** [TsingZ0/PFL-Non-IID#9]
Hi TsingZ0, I fork your code and run the default code "main.py" with mnist dataset, under iid setting, but I am confused that why the "Averaged Test Accuracy" is increasing slowly. My previous experiments on other code libraries have shown that accuracy is usually more than 90% at around 20 rounds. Can you explain it?

**Comment by TsingZ0** [comment]
Did you check the hyper-parameter settings?

**Comment by Sherrylife** [comment]
the hyper-parameter settings is the same as in the main.py file in the repository, and I didn't make any changes.

**Comment by Sherrylife** [comment]
When I run the FedAvg in other code libraries, (e.g. https://github.com/JedMills/MTFL-For-Personalised-DNNs), my experience is that learning rate adjusted to 0.01 can achieve communication over 20 rounds with 90% accuracy, but I am curious that this does not seem to be possible in your code base.

**Comment by Sherrylife** [comment]
Hi,   
I pulled your latest code and found that my problem was solved. Although I don't know what caused it, thank you all the same.

**Comment by TsingZ0** [comment]
I have changed the default hyper-parameter settings in the latest version.
